stable/handbrake/3.0.0/questions.yaml

```diff
@@ -51,14 +69,32 @@ questions:
       $ref:
         - "definitions/timezone"
 
+  # PUID
```

Comment:
We don't need not want these comments with obvious things, besides this portion is the same for every container and your change brakes copy-paste scripts.